lib/schema.coffee

```diff
@@ -1,7 +1,44 @@
+validTypes = [
+  'headline'
+  'h1'
+  'h2'
+  'h3'
+  'h4'
+  'h5'
+  'h6'
+  'text'
+  'list'
+  'table'
+  'code'
+  'image'
+  'video'
+  'audio'
+  'article'
+  'quote'
+  'location'
+  'cta'
+  'interactive'
+  'placeholder'
+  'unknown'
+]
+
+isValidType = (type, solving = false) ->
+  if typeof type is 'object'
+    type = type.type
+  return false unless typeof type is 'string'
```

Comment:
Should throw. It is an error by the programmer to pass in something which not a block / type.